docs/design/fabric-v2+/fpc-management.md

```diff
@@ -59,3 +59,32 @@ indicating that the chaincode enclave is ready to endorse transaction proposals.
 
 Key distribution commands are not supported in the initial version of FPC.
 Users must aware that each FPC chaincode runs in a single enclave at a designated peer.
+
+
+## FPC Endorsement Policies
```

Comment:
added the reference!

Reply:
> Not sure whether we should mention it somewhere, but another constraint is now that of course invokes _must_ but directed to that particular peer. (No idea, though, whether the cli and/or SDK is already smart enough to look at endorsement policy and automatically pick sufficient and correct peers to issue request to ...)

I guess not! The CLI by default tries to invoke the peer on localhost (I assume), otherwise the endorsing peers need to be defined using `--peerAddresses stringArray      The addresses of the peers to connect to`.